.github/workflows/kustomize-build-ci.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: kustomize-build-ci
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'kustomize/**'
+      - '.github/workflows/kustomize-build-ci.yml'
+  pull_request:
+    paths:
+      - 'kustomize/**'
+      - '.github/workflows/kustomize-build-ci.yml'
+jobs:
+  job:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: kustomize build base
+        run: |
+          cd kustomize/
+          kubectl kustomize .
+      - name: kustomize build cymbal-branding
+        run: |
+          cd kustomize/
+          kustomize edit add component components/cymbal-branding
+          kubectl kustomize .
+      - name: kustomize build google-cloud-operations
+        run: |
+          cd kustomize/
+          kustomize edit add component components/google-cloud-operations
+          kubectl kustomize .
+      - name: kustomize build memorystore
+        run: |
+          cd kustomize/
+          kustomize edit add component components/memorystore
```

Comment:
**Question:**
Each step is adding a Component and then running Kustomize-building.
My question: Are they building on top of each other?
For instance, does the `kustomize build google-cloud-operations` step include the `cymbal-branding` Component?
Reason I ask: I would rather not build on top of each. I think we should test each Component on its own. I think promising users that all the Components ALL components will work together is a big commitment. I think the CUJ we should 100% support and prioritize is "each Component can be deployed on its own".

Reply:
Ah ah, you just read my mind and my screen actually! Aligned with you, 100% agreeed on that.

I will work on the draft mode to include this:
- Do not repeat the steps, but have loop in bash browsing the subfolders in `components` - with that, as soon as a new component is added, no need to update the CI definition
- Each test will be done alone, just 1 component at a time
- The last test will combined all the components, to have kind of a "test them all"

Reply:
Done! Let me know what you think! ;)